evm/p2p/eth.py

```diff
@@ -0,0 +1,206 @@
+import logging
+from typing import Any, cast, Dict, List, Union
+
+import rlp
+from rlp import sedes
+
+from eth_utils import encode_hex
+
+from .constants import MAX_HEADERS_FETCH
+from evm.rlp.headers import BlockHeader
+from evm.rlp.receipts import Receipt
+from evm.rlp.transactions import BaseTransaction
+from evm.p2p.exceptions import HandshakeFailure
+from evm.p2p.les import HashOrNumber
```

Comment:
Seems like this should get moved to a separate `sedes` file rather than it living in `les` now that it's being used in multiple places.